tests/unit/EntityMenu.spec.ts

```diff
@@ -499,8 +499,9 @@ describe.skip('Entity Menu - Request AGM Extension click tests', () => {
       mixins: [{ methods: { isAllowed: () => true } }],
       propsData: { businessId: 'BC1234567' }
     })
-
     vm = wrapper.vm
+
+    vi.spyOn(vm, 'enableAgmLocationChg', 'get').mockReturnValue(CorpTypeCd.BENEFIT_COMPANY)
```

Comment:
OK, this is less of a hack than the other version.

I was thinking that lines 486-487 could look like this:
```
vi.spyOn(utils, 'GetFeatureFlag').mockImplementation(flag => {
 if (flag === 'enable-agm-extension') return True
 if (flag === 'supported-agm-location-chg-entities') return 'BEN'
})
```

This might not work or might not fix the root cause. But I think the error you were seeing is because GetFeatureFlag was returning a boolean, and then some code was trying to use it like an array (eg, true.include(...)).

Reply:
Ohhh I see. This worked Sev! Makes a lot of sense, thank you so much.

Reply:
Small correction of what I wrote:

> some code was trying to use it like an array

or *string**.